vscode-ext/src/api.ts

```diff
@@ -0,0 +1,267 @@
+import * as vscode from 'vscode';
+import * as path from 'node:path';
+import * as fs from 'node:fs';
+import { ChildProcessWithoutNullStreams, spawn } from 'child_process';
+import { globalContext } from './extension';
+const util = require('node:util');
+const exec = util.promisify(require('node:child_process').exec);
```

Comment:
These two should no longer be needed, right?

Reply:
yes you are right, thanks!